src/base/dof_map_constraints.C

```diff
@@ -2281,23 +2281,21 @@ std::string DofMap::get_local_constraints(bool print_nonlocal) const
   os << "Node Constraints:"
      << std::endl;
 
-  for (const auto & pr : _node_constraints)
+  for (const auto & [node, pr] : _node_constraints)
```

Comment:
It's a shame there's no movement on nested structured bindings.  I found [a "flatten" function](https://www.reddit.com/r/cpp_questions/comments/eto1rh/nesting_structured_bindings/) that works around the problem for some declarations but I don't think it'd be usable in a range for loop.